src/app/FeedbackForm.js

```diff
@@ -38,51 +113,77 @@ const StyledButton = styled.button`
   font-size: 1.3em;
 `;
 
-export default function FeedbackForm({ questions, onSubmit, history }) {
+export default function FeedbackForm({ questions, handleSubmit, history }) {
+  function handleFormSubmit(event, questions, history) {
+    event.preventDefault();
+    const form = event.target;
+    const answers = questions.map(
+      question => form[`q-${question.id + 1}`].value
+    );
+    console.log('Answers:', answers);
+    handleSubmit(answers, history);
+  }
+
   return (
-    <StyledForm onSubmit={event => onSubmit(event, history)}>
-      <StyledQuestionContainer>
-        <StyledQuestion>
-          Q{questions[0].question_id + 1}. {questions[0].question_text_en}
-        </StyledQuestion>
-        <StyledResponse>
-          <div>
-            <label>
-              <input type="radio" name="res" value="1" />
-              strongly disagree
-            </label>
-          </div>
-
-          <div>
-            <label>
-              <input type="radio" name="res" value="2" />
-              disagree
-            </label>
-          </div>
-
-          <div>
-            <label>
-              <input type="radio" name="res" value="3" defaultChecked={true} />
-              neutral
-            </label>
-          </div>
-
-          <div>
-            <label>
-              <input type="radio" name="res" value="4" />
-              agree
-            </label>
-          </div>
-
-          <div>
-            <label>
-              <input type="radio" name="res" value="5" />
-              strongly agree
-            </label>
-          </div>
-        </StyledResponse>
-      </StyledQuestionContainer>
-      <StyledButton>Add Feedback</StyledButton>
-    </StyledForm>
+    <Grid>
+      <StyledForm
+        id="addFeedback"
+        onSubmit={event => handleFormSubmit(event, questions, history)}
+      >
+        {questions.map(question => (
+          <StyledQuestionContainer key={question.id}>
+            <StyledQuestion>
+              Q{question.id + 1}. {question.question_text_en}
+            </StyledQuestion>
+
+            <StyledRadioButtons>
```

Comment:
maybe you can make a component for this where you iterate to render `<label>` and `<input>` inside of `<StyledRadioButtons>`